### DISCUSSION TITLE
predict_model not working in a new python console when model pkl is loaded

### DISCUSSION THREAD

**Discussion by biased-ai** [pycaret/pycaret#3363] [General]
I trained multiple models on pycaret for 700 time series and saved it using save_model. When I load the model again into a new python environment( Python same version) , the predict_model does not work it returns an error, estimator does not have an attribute decision_function.

Is it possible, to load models in a new environment ( same version where time series were trained)?

P.S: my time series have been trained on the same pycaret version.

If i score in the existing console itself the code works perfectly fine. Dont know where am I going wrong

**Comment by Yard1** [comment]
What pycaret version are you using?

**Comment by biased-ai** [comment]
pycaret 3.0.0

On Thu, 9 Mar 2023 at 10:18 AM, Antoni Baum ***@***.***>
wrote:

> What pycaret version are you using?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pycaret/pycaret/discussions/3363#discussioncomment-5249577>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEVWEZQBKGDO5TA543FOCS3W3FOKFANCNFSM6AAAAAAVQ5JVLU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Best Regards,
Manas Manoj
